less/dropdowns.less

```diff
@@ -34,7 +34,7 @@
   padding: @availity-menu-padding-vertical (@availity-menu-padding-vertical + @availity-menu-padding-horizontal);
   text-transform: uppercase;
   color: @white;
-  background-color: #02a17c;
+  background-color: #157461;
```

Comment:
Can you change this to use `@green` (or `@secondary-green`... since they are the same 😕 ) to avoid having the hex duplicated here.

Reply:
Good point, I had just done a find/replace on the whole project.